src/ILVerification/src/ILVerification.csproj

```diff
@@ -301,6 +301,9 @@
     <Compile Include="..\..\Common\src\TypeSystem\Common\TypeSystemConstraintsHelpers.cs">
       <Link>TypeSystem\Common\TypeSystemConstraintsHelpers.cs</Link>
     </Compile>
+    <Compile Include="..\..\Common\src\TypeSystem\Interop\IL\MarshalUtils.cs">
```

Comment:
Hmm, interesting, I originally put it there because the test was complaining about the missing symbol but apparently the base tests are passing even after I removed the line - I guess I most likely incorrectly assigned the failure to the test suite in the interleaved parallel test build console output.